block/manager.go

```diff
@@ -279,23 +278,5 @@ func (m *Manager) applyBlockCallback(event pubsub.Message) {
 
 // getLatestBatchFromSL gets the latest batch from the SL
 func (m *Manager) getLatestBatchFromSL(ctx context.Context) (*settlement.ResultRetrieveBatch, error) {
-	var resultRetrieveBatch *settlement.ResultRetrieveBatch
-	var err error
-	// Get latest batch from SL
-	err = retry.Do(
-		func() error {
-			resultRetrieveBatch, err = m.settlementClient.RetrieveBatch()
-			if err != nil {
-				return err
-			}
-			return nil
-		},
-		retry.LastErrorOnly(true),
-		retry.Context(ctx),
-		retry.Attempts(1),
-	)
-	if err != nil {
-		return resultRetrieveBatch, err
-	}
-	return resultRetrieveBatch, nil
+	return m.settlementClient.RetrieveBatch()
 }
```

Comment:
this function could be inlined, it seems

Reply:
approved anyway

Reply:
right. done in other PR